packages/scandipwa/src/component/CheckoutOrderSummary/CheckoutOrderSummary.style.scss

```diff
@@ -17,6 +17,12 @@
 .CheckoutOrderSummary {
     $divider-border: 1px solid var(--input-border-color);
 
+    @include mobile {
+        > .Loader {
+            display: none;
+        }
+    }
```

Comment:
Or maybe we shouldn't render this loaded at all? :)

Reply:
it's not rendered with display: none. i think there's smth like "isMobile" prop in javascript but using it yields the same result as these CSS rules. i agree that i don't see the utility in this loader, but it was requested by QA [here](https://github.com/scandipwa/scandipwa/issues/4078). @carinadues what do you think, should we maybe just remove it altogether for all devices?